src/models/Expense.model.js

```diff
@@ -1,9 +1,47 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
   // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.INTEGER,
```

Comment:
Consider using `DataTypes.FLOAT` or `DataTypes.DECIMAL` for the `amount` field if you need to handle fractional currency values. Using `DataTypes.INTEGER` might not be suitable for all currency calculations.